raiden/network/rpc/client.py

```diff
@@ -126,7 +127,7 @@ def format_data_for_call(
         to: address = b'',
         value: int = 0,
         data: bytes = b'',
-        startgas: int = GAS_PRICE,
+        startgas: int = GAS_LIMIT,
         gasprice: int = GAS_PRICE):
```

Comment:
In another occasion this is `gas_price` which I find nicer. Whichever we choose, we should be consistent.